### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by Gautam2305**
- Added two types of avatar components : one with user image and another with default avatar
- Both the types have three types of avatar sizes : small, medium & large.
- Gave border-radius of 50% and white border.
- Sizes for avatars: Large - 10rem; Medium - 6.25rem; Small - 3rem;
- added loading lazy for images as it will not be the first thing user sees in the documentation website
Screenshot (76)

### CODE REVIEW

File: avatar.html [hunk 1]
```diff
@@ -9,9 +9,16 @@
...
 </head>
 <body>
-    <img class="avatar sm" src="assets/official_pic.svg" placeholder="GK" />
-    <img class="avatar md" src="assets/official_pic.svg" placeholder="GK" />
-    <img class="avatar lg" src="assets/official_pic.svg" placeholder="GK" />
-
+    <div>
+    <img loading="lazy" class="avatar avatar-lg-size" src="assets/_official_pic.jpg" alt="Avatar Picture"/>
+    <img loading="lazy" class="avatar avatar-md-size" src="assets/_official_pic.jpg" alt="Avatar Picture"/>
+    <img loading="lazy" class="avatar avatar-sm-size" src="assets/_official_pic.jpg" alt="Avatar Picture"/>
```
Comment (ttahm3d) [inline avatar.html]: The word size is may not be required.

File: stylesheets/Components/avatar.css [hunk 2]
```diff
@@ -1,4 +1,28 @@
...
-    
-}
\ No newline at end of file
+    border-radius: 50%;
+    border: 2px solid white;
+}
+.avatar-lg-size{
+    width: 10rem;
+    height: 10rem;
+    font-size: 3rem;
```
Comment (ttahm3d) [inline stylesheets/Components/avatar.css]: I'm not 100% sure, but I dont think font-size is required here. If you have any specific reason why you are using it, please state

**Review by ttahm3d — COMMENTED** [review]
Check comments and change accordingly.

File: avatar.html (hunk 1 above)
Comment (Gautam2305) [reply avatar.html]: Okay. Will update it right away.

File: stylesheets/Components/avatar.css (hunk 2 above)
Comment (Gautam2305) [reply stylesheets/Components/avatar.css]: Yeah. Actually I had another avatar with text inside it at first & i had given this class to them as well. I decided not to go forward with that but forgot to delete this styling. I will resolve it now. 
Thank you for your feedback❤️.

**Review by ttahm3d — APPROVED** [review]
Good to merge

**Pull request merged**